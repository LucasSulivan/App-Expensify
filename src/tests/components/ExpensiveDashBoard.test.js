import React from "react";
import { shallow } from "enzyme";
import ExpensiveDashBoard from '../../Components/expensiveDashBoard';

test('should render expensivedashboard page',()=>{
  const wrapper = shallow(<ExpensiveDashBoard />)
  expect(wrapper).toMatchSnapshot();
});