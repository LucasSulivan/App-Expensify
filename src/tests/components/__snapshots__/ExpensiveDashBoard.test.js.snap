// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render expensivedashboard page 1`] = `
<div>
  <Connect(ExpensesSummary) />
  <Connect(ExpenseList) />
  <Connect(ExpenseListFilter) />
</div>
`;
